tardis/io/model/readers/cmfgen_deprecated.py

```diff
@@ -0,0 +1,91 @@
+from tardis.io.model.readers.util import read_csv_isotope_mass_fractions
```

Comment:
do we know what's going on with this file?

Reply:
It's the old reader, it actually produces a result but it's deprecated. We can clean it out once we are happy the new reader works as intended.